File: java/Java_Work/sum_ave/src/sum_ave/SumAve.java
package sum_ave;

public class SumAve {
	private int sum;
	private int ave;
	
	public int numSum(int arr[]) {
		for (int i = 0; i < arr.length; i++) {
			sum += arr[i];
		}
		return sum;
	}
	
	public int numAve(int arr[]) {
		for (int i = 0; i < arr.length; i++) {
			sum += arr[i];
		}
		ave = sum / arr.length;
		return ave;
	}
}

File: C_sharp/Class_practice_20210208/Class_practice_2/SumAve.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Class_practice_2
{
    class SumAve
    {
        private int sum;
        private int ave;

        public int getSum(int[] arr)
        {
            sum = 0;
            for (int i = 0; i < arr.Length; i++)
            {
                sum += arr[i];
            }
            return sum;
        }

        public int getAve(int[] arr, int sum)
        {
            ave = sum / arr.Length;
            return ave;
        }
    }
}
